website/docs/guides/Programmatic-API-Key-upgrade-guide-1.10.0.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "mongodbatlas"
+page_title: "Upgrade Guide for Terraform MongoDB Atlas Provider Programmatic API Key Resource in v1.10.0"
+sidebar_current: "docs-mongodbatlas-guides-Programmatic-API-Key-upgrade-guide"
+description: |-
+MongoDB Atlas Provider : Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider: Programmatic API Key Upgrade Guide in v1.10.0
+In Terraform MongoDB Atlas Provider v1.10.0, some improvements were introduced which mainly focus on the MongoDB Atlas Programmatic API Keys (PAK) handling. This guide will help you to transition smoothly from the previous version which this resource was first released (v1.8.0) to the new version (v1.10.0).
+
+For comprehensive Upgrade Guide on all v1.10.0 modifications see [here](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/1.10.0-upgrade-guide). 
+
+Remember, your scripts will still work with deprecated features for now, but it's best to upgrade as soon as possible to benefit from the latest enhancements. Code removal is planned for v1.12.0 at which point prior PAK workflow will no longer function.
+
+Lastly, in MongoDB Atlas, all PAKs are Organization API keys. Once created, a PAK is linked at the organization level with an 'Organization Member' role. However, these Organization API keys can also be assigned to one or more projects within the organization. When a PAK is assigned to a specific project, it essentially takes on the 'Project Owner' role for that particular project. This enables the key to perform operations at the project level, in addition to the organization level. The flexibility of PAKs provides a mechanism for fine-grained access and control, once their functioning is clearly understood.
```

Comment:
This is such a hard thing to explain.  I really wish our API was a bit more like our UI.  But since it's not and with the sames we have I think we could make this a bit clearer. Gave it a try here:
```suggestion
Lastly, in MongoDB Atlas, all PAKs are Organization API keys. Once created, a PAK is linked at the organization level with at minimum the 'Organization Member' role. However, these Organization API keys can also be assigned to one or more projects within the organization. When a PAK is assigned to a specific project, it essentially becomes a Project level PAK for that particular project. This enables the key to perform operations at the project level with project specific roles, in addition to the organization level. The flexibility of PAKs provides a mechanism for fine-grained access and control, once their functioning is clearly understood.  
```